assets/js/editor/editor.js

```diff
@@ -26,7 +26,7 @@ const CONSOLE_HIDE_BUTTON_ID = "console-btn--hide";
 const CURRENT_SUBMITS_DATA_KEY = "data-current-submissions";
 const MAX_SUBMITS_DATA_KEY = "data-submission-limit";
 const NO_SUBMISSION_LIMIT = -1;
-
+const EDITOR_LOOK_FOR_CHANGES_OFFSET = 1000;
```

Comment:
Note to self: check what happens if the user presses "Save" in between the offset.

Reply:
Well nothing happens. Pressing "Save" during that time means, that the save button won't be activated by the debounce function, because there are no changes.